HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectDoor.cs

```diff
@@ -0,0 +1,114 @@
+﻿using GraphicalEditor.Enumerations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Shapes;
+
+namespace GraphicalEditor.Models.MapObjectRelated
+{
+    public class MapObjectDoor
+    {
+        private Rectangle _rectangle;
+        private MapObjectMetrics _mapObjectMetrics;
+        private MapObjectDoorOrientation _mapObjectDoorOrientation;
+        private double _XShift;
+        private double _YShift;
+
+        private readonly double DOOR_WIDTH = 45;
+        private readonly double DOOR_HEIGHT = 10;
+
+        public MapObjectDoor(MapObjectDoorOrientation mapObjectDoorOrientations, double XShift = 0, double YShift = 0)
+        {
+            MapObjectDoorOrientation = mapObjectDoorOrientations;
+            this.XShift = XShift;
+            this.YShift = YShift;
+        }
+
+        public Rectangle GetDoor()
```

Comment:
Change proposal: Replace the GetDoor, GetDoorWidth, and GetDoorHeight methods with properties. I believe they don't violate any of the usage guidelines outlined in the [Properties vs. Methods section here](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/bzwdh01d(v=vs.71)?redirectedfrom=MSDN#cpconpropertyusageguidelinesanchor1), especially GetDoorWidth and GetDoorHeight. GetDoor might requite some further consideration since it is a somewhat more expensive operation.

Reply:
I'll convert GetDoorWidth and GetDoorHeight to properties since it is a good suggestion but will leave GetDoor since I don't like long properties especially if they rely on other methods.

Reply:
Sounds good, I agree that GetDoor is better left as a method.